checkstyle/suppressions.xml

```diff
@@ -183,6 +183,8 @@
               files="StreamsPartitionAssignor.java"/>
     <suppress checks="JavaNCSS"
               files="EosBetaUpgradeIntegrationTest.java"/>
+    <suppress checks="StaticVariableName"
```

Comment:
Checkstyle won't allow you to have a letter and number next to each other, but in the case of `E2E_LATENCY` I feel that's a lot easier to understand that `E_2_E_LATENCY`